modyn/trainer_server/internal/dataset/data_utils.py

```diff
@@ -33,6 +33,7 @@ def prepare_dataloaders(
         transform (list[str]): List of serialized torchvision transforms for the samples, before loading.
         storage_address (str): Address of the Storage endpoint that the OnlineDataset workers connect to.
         selector_address (str): Address of the Selector endpoint that the OnlineDataset workers connect to.
+        return_weights (bool): if the dataloader must return the weights
```

Comment:
this is outdated, right? because this is the only addition in this file

Reply:
Yep, sorry